cobalt/loader/net_fetcher_test.cc

```diff
@@ -139,7 +139,6 @@ TEST_F(NetFetcherTest, DISABLED_LocalFetch) {
   run_loop.Run();
 
   std::string loaded_text = fetcher_handler_mock.data();
-  printf("%s\n", loaded_text.c_str());
```

Comment:
I'd assume we want a LOG instead?

Reply:
I'm not sure what's the goal of this test. I would expect some checks for this content. @aee-google Can you suggest what shall we do here? LOG(INFO)?

Reply:
Umm this looks to be a debug print